Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM wordpress:5.3
+
+COPY . /var/www/html/wp-content/plugins/woocommerce
```

Comment:
It is true that currently, we need to rebuild the container if we make a change in the code. I don't see it as a very big issue at the moment but it will be easier to work if we would mount a volume instead. 

I tried to do this yesterday but it turned out to be not a very trivial task. There is a permission issue in place which is well described [here](https://stackoverflow.com/questions/50325494/how-can-i-change-permission-of-mounted-volumes-in-docker-compose-yml-from-the-do) and [here](https://stackoverflow.com/questions/29245216/write-in-shared-volumes-docker/29251160#29251160). AFAIK, it is a bit more straightforward how to make it work in Linux but more complicated in macOS due to the fact that Docker runs on a virtual machine on mac.

I will need to take some time to research this further. What do you think if I work on it in the next stage of this project after we merge what we have as-is?

Reply:
Ah ok, no worries, if it's not trivial then I guess it should not block rest of the other work.

Reply:
I created an issue to document the task: https://github.com/woocommerce/woocommerce/issues/25274